pkg/kubelet/dockertools/manager.go

```diff
@@ -1830,8 +1830,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, runningPod kubecontainer.Pod, pod
 		}
 
 		// Killing phase: if we want to start new infra container, or nothing is running kill everything (including infra container)
-		err = dm.KillPod(pod, runningPod)
-		if err != nil {
+		if err = dm.KillPod(pod, runningPod); err != nil {
```

Comment:
(nit) You don't shadow the outer variable here, but do below. I don't think it matters in either case, but is there any reason to use one over the other?